packages/vue/src/components/atoms/SfHeading/SfHeading.html

```diff
@@ -0,0 +1,20 @@
+<header class="sf-heading">
+  <template v-if="level===1">
+    <h1 class="sf-heading__title"><slot>
+      {{title}}
+    </slot></h1>
+  </template>
+  <template v-if="level===2">
+    <h2 class="sf-heading__title"><slot>
```

Comment:
can be refactored to dynamic component

Reply:
like this >>https://vuejs.org/v2/guide/render-function.html<<